docs/index.html

```diff
@@ -747,6 +748,41 @@ <h4 id="window-contents">Window contents</h4>
       </div>
     </section>
 
+    <section class="component">
+      <h3 id="status-bar">Status Bar</h3>
+      <div>
+        <blockquote>
+          I don't have a copy of this book and I don't know if it talks about status bars.
+
+          <footer>
+            &mdash; Microsoft Windows User Experience p. 0
+          </footer>
+        </blockquote>
+
+       <p>To render a status bar, use a <code>table</code> element with
+         the <code>status-bar</code> class. The <code>td</code> elements
+         within it become status bar panels. The second through last panels
+         will be sized their contents; the first will fill in the rest of the
+         space.
+
+         <div class="example" style="width:400px">
+            <table class="status-bar">
+              <td>12 object(s)
+              <td>5.47MB
+              <td>&#x1f4bb; My Computer
+            </table>
+          </div>
+          <details>
+            <summary>Show code</summary>
+            <pre><code>&lt;table class="status-bar">
```

Comment:
```suggestion
            <pre><code>&lt;table class="status-bar"&gt;
```
Nitpick: Style consistency.

Reply:
Just read through the other comments however, might not need to apply this 😄 Depending on how `index.html` is being handled.